frontend/server/src/Controllers/User.php

```diff
@@ -2419,6 +2420,7 @@ public static function getUserRankInfo(
             'rank' => 0,
             'name' => strval($identity->name),
             'problems_solved' => 0,
+            'author_ranking' => 0,
```

Comment:
```suggestion
            'author_ranking' => null,
```

tal vez? usemos `null` en vez de números "mágicos".